sdk/samples/azidentity/example_SDKV1_test.go

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentitysamples
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
+
+	"github.com/jongio/azidext/go/azidext"
+)
+
+// Please note that the examples in this file are using the Azure SDK for Go V1 code base, along
+// with azidentity package from the Azure SDK for Go V2.
+// The adapter in the azidext package provides a simple way to integrate azidentity credentials
+// as authorizers for the V1 code base.
+
+const (
+	groupName = "samplegroup"
+)
+
+// Environment variables required for EnvironmentCredential to work and/or DefaultAzureCredential
+var (
+	clientID       = os.Getenv("AZURE_CLIENT_ID")
+	clientSecret   = os.Getenv("AZURE_CLIENT_SECRET")
+	subscriptionID = os.Getenv("AZURE_SUBSCRIPTION_ID")
+	tenantID       = os.Getenv("AZURE_TENANT_ID")
+)
+
+var (
+	location  = os.Getenv("AZURE_LOCATION")
+	userAgent = "azidentitysample"
+)
+
+// Example for using the DefaultAzureCredential through the NewDefaultAzureCredentialAdapter and assigning the credential to the
+// SDK V1 authorizer.
+func getGroupsClientWithDefaultAzureCredential() resources.GroupsClient {
+	groupsClient := resources.NewGroupsClient(subscriptionID)
+	a, err := azidext.NewDefaultAzureCredentialAdapter(nil)
```

Comment:
I would add a comment here explaining what this is doing and how it relates to `auth.NewAuthorizerFromEnvironment()`

Reply:
Also, isn't this missing the scopes?

Reply:
Done, although I will say that apparently it didnt need it. The example ran successfully even before I added the scopes

Reply:
Ah yes you're correct.  `NewDefaultAzureCredentialAdapter()` adds the management scope if options are nil.  So you can leave this as nil.  Probably worth adding a comment here explaining this.

Reply:
Done and comment has been added